player/public/coreplugins/org.ekstep.ecmlrenderer-1.0/renderer/qspatch.js

```diff
@@ -390,5 +390,70 @@ var qspatch = {
             'image': data.image ? data.image : undefined,
             'audio': data.audio ? data.audio : undefined,
         })
+    },
+    //Make default font size for question options
+    setDefaultFontSize: function(data){
+      if(data.stage['org.ekstep.questionset']){
+        try {
+          var questionSetData = data.stage['org.ekstep.questionset'];
+          _.each(questionSetData['org.ekstep.question'],function(item,key){
+              var questionData = JSON.parse(item.data.__cdata);
+              questionData.question.text = qspatch.changeFontSize(questionData.question);
+              item.data.__cdata = JSON.stringify(questionData);
+              item = qspatch.questionFontSizeChange(item);
+              questionSetData['org.ekstep.question'][key] = item;
+          });
+          // Renderer.theme = new ThemePlugin(questionSetData);
+        } catch (e) {
+            console.log(e);
+        }
+      }
+    },
+    questionFontSizeChange: function(item){
+      var questionItem = item;
+      var questionOptionsData = JSON.parse(item.data.__cdata);
+
+      switch (item.pluginId) {
+        case 'org.ekstep.questionunit.mcq':
+        case 'org.ekstep.questionunit.sequence':
+              questionOptionsData.options = qspatch.optionsTextFontChange(questionOptionsData.options,item.type);
+              questionItem.data.__cdata = JSON.stringify(questionOptionsData);
+              break;
+        case 'org.ekstep.questionunit.mtf':
+              questionOptionsData.option.optionsLHS = qspatch.optionsTextFontChange(questionOptionsData.option.optionsLHS,item.type);
+              questionOptionsData.option.optionsRHS = qspatch.optionsTextFontChange(questionOptionsData.option.optionsRHS,item.type);
+              questionItem.data.__cdata = JSON.stringify(questionOptionsData);
+              break;
+        default: break;
+
+      }
+      return questionItem;
+    },
+    optionsTextFontChange: function(options,type){
```

Comment:
Change the function name
"changeOptionsFontSize"

Reply:
Done